src/Extractor.cc

```diff
@@ -5113,8 +5109,8 @@ namespace insur {
       volumes.push_back(vol[xml_PixelModuleChip]);
       volumes.push_back(vol[xml_PixelModuleDeadAreaRight]);
      if(module.numSensors()==2){
-        volumes.push_back(vol[xml_PixelModuleDeadAreaBackOfCentre]);
         volumes.push_back(vol[xml_PixelModuleDeadAreaFrontOfCentre]);
```

Comment:
Is the order relevant here?

Reply:
No, it's irrelevant, but everywhere else in the code `Front` goes before `Back`.  
Stylistically it's more consistent this way.